### ISSUE TITLE
Public Access Enhancements for TRAJAN Website

### ISSUE THREAD

**Issue by xyzerobtc** [zeroauthority-dao/Overview#89]
Public Access Enhancements for TRAJAN Website

Scope of Work and Requirements: 

**Objective:**
- Enhance the TRAJAN website to facilitate improved user engagement by making profiles and endorsements accessible to the public. The enhancements aim to create a more open and transparent platform, fostering a sense of community and trust among users.

**Profile Visibility:**
- Implement a user-friendly profile interface allowing users to customize and manage their profiles.
- Enable the option for users to choose the visibility level of their profiles (public, private, or limited visibility).
- Ensure that essential user information such as skills, experience, and endorsements are prominently displayed.

**Public Profile Access:**
- Modify the website's privacy settings to allow public access to user profiles.
- Ensure that public profiles display key information while respecting user privacy preferences.

**Notification System:**
- Integrate a notification system to inform users about profile views, new endorsements, and connection requests.
- Provide users with the ability to customize notification preferences.

**User Interface (UI) and Experience (UX) Improvements:**
- Optimize the website's UI/UX to accommodate the new profile and endorsement features seamlessly.
- Conduct usability testing to identify and address any potential issues in the user journey.

**User Documentation:**
- Develop comprehensive user documentation explaining the new profile and endorsement features.
- Provide step-by-step guides for users to manage their profiles, endorsements, and privacy settings effectively.

**Comment by xyzerobtc** [comment]
Add endorsements to the landing page. 

![image](https://github.com/zeroauthority-dao/Overview/assets/16929732/27905bb4-eb7b-475b-8656-1738625742e1)

**Comment by xyzerobtc** [comment]
Add endorsements to the landing page. 

![image](https://github.com/zeroauthority-dao/Overview/assets/16929732/d251748e-8be2-4bf4-8d5f-ca30a730995b)

**Comment by hozzjss** [comment]
hey @xyzerobtc do you need the landing page to be the home page that is shown when people are logged in? I have made all the APIs public so what remains are the front end updates

**Comment by xyzerobtc** [comment]
@hozzjss how difficult (or time wise) would it be to move connect to the page after landing page? 

![trajan](https://github.com/zeroauthority-dao/Overview/assets/16929732/16515c2a-7e70-43c5-9acc-7535d8b626f5)

**Comment by xyzerobtc** [comment]
> hey @xyzerobtc do you need the landing page to be the home page that is shown when people are logged in? I have made all the APIs public so what remains are the front end updates

@hozzjss yes, that is what i'm asking you - how difficult or how much time would it take, and how much $stx can we give you
